vent/menu.py

```diff
@@ -35,11 +35,12 @@ def onStart(self):
         curses.mousemask(0)
         self.paths.host_config()
         version = Version()
-        quit_s = "\t"*6 + "Press ^Q to quit"
+        quit_s = "\t"*4 + "Press ^Q to quit"
+        tab_esc = "\t"*4 + "Press TAB to close popup"
```

Comment:
let's specify this as `menu popup` since regular popup's have an `ok/cancel` which are triggered by enter, not tab.